lti_consumer/lti_1p1/contrib/django.py

```diff
@@ -0,0 +1,125 @@
+"""
+This module provides functionality for rendering an LTI embed without an XBlock.
+"""
+
+from __future__ import absolute_import, unicode_literals
```

Comment:
These aren't needed for python 3

Reply:
This is resolved in the most recent commit